build.gradle.kts

```diff
@@ -37,6 +37,11 @@ dependencies {
 
 	implementation("net.bytebuddy:byte-buddy:1.17.6")
 
+	compileOnly("org.projectlombok:lombok:1.18.34")
+	annotationProcessor("org.projectlombok:lombok:1.18.34")
+	testCompileOnly("org.projectlombok:lombok:1.18.34")
+	testAnnotationProcessor("org.projectlombok:lombok:1.18.34")
```

Comment:
@MockBukkit/maintainers what are your opinion about this?
I don't see any problem in using lombok, but i think it should be a community decision :)

Reply:
Oh, you wrote the same question, I'm personally against it, but it does not matter that much. From my point of view getters and setters should be avoided anyhow, but that might not be possible as we are faking a whole API, which happily uses those types of methods